### ISSUE TITLE
Fan speed tooltip

### ISSUE THREAD

**Issue by ErikDeBruijn** [Ultimaker/Cura#263]
Not sure, but could it be that minimum and maximum were swapped in this explanation?

![image](https://cloud.githubusercontent.com/assets/191236/9306227/3ad2bdf8-44f7-11e5-96a6-4865e57f383e.png)

I'm a big "fan" of tooltips, however :)

**Comment by Xisle** [comment]
While on the topic of tooltips..

The tooltips for Machine Settings have an issue with the display timer. It looks like the mouseEntered() is being handled before mouseExited() and mouseEntered() isn't stopping the previous Timer.

**Comment by Xisle** [comment]
https://www.youtube.com/watch?v=TJU-aoBvDw0&feature=youtu.be

**Comment by BagelOrb** [comment]
@Xisle please submit a new issue. These two are unrelated.

@ErikDeBruijn the description is correct, though hard to understand. A later cannot take less time than the minimal layer time, at which points the fan will be at Max speed. For  layers taking longer the fan speed reduces, until a layer takes longer than "minimal layer time minimal fan speed".

I should maybe rephrase the tooltip...

**Comment by Xisle** [comment]
I was already working on it ;)
